scss/bootstrap/overrides/style/_icons.scss

```diff
@@ -1,4 +1,5 @@
 a.crm-i:hover,
-a.fa:hover {
+a.fa:hover,
+a.helpicon {
```

Comment:
I know this looks like the most convenient place to add this rule, but overriding CiviCRM rules need to be placed here:
[/scss/bootstrap/overrides/crm](https://github.com/compucorp/org.civicrm.shoreditch/tree/7b307de495cdc0c9c9b4a038ff71d8953beb03e2/scss/bootstrap/overrides/crm)

This file should only contain overriding rules for Bootstrap (.btn, .panel, etc) and FontAwesome (.fa, .fa-warning). That `a.crm-i:hover` is wrong too.